portal-web/test/functional/com/liferay/portalweb/tests/enduser/publications/PublicationsAssetList.testcase

```diff
@@ -86,4 +86,52 @@ definition {
 		}
 	}
 
+	@description = "Assert that is possible to delete a Asset List within Publications"
+	@priority = 3
+	test CanDeleteAssetList {
+		task ("Create a manual asset list") {
+			JSONAssetlist.addManualAssetListEntry(
+				groupName = "Guest",
+				title = "Manual Asset List Test");
+		}
+
+		task ("Create a dynamic asset list") {
+			JSONAssetlist.addDynamicAssetListEntry(
+				groupName = "Guest",
+				title = "Dynamic Asset List Test");
+		}
+
+		task ("Add new Publication and use it") {
+			JSONPublications.addPublication(publicationName = "Publication Name");
+
+			JSONPublications.selectPublication(publicationName = "Publication Name");
+		}
+
+		task ("Delete both asset lists") {
+			AssetListsAdmin.openAssetListsAdmin(siteURLKey = "guest");
+
+			AssetListsAdmin.deleteEntry(assetListTitle = "Manual Asset List Test");
+
+			AssetListsAdmin.deleteEntry(assetListTitle = "Dynamic Asset List Test");
+		}
+
+		task ("Assert the taglib that displays when there are no entries displays") {
+			AssertElementPresent(locator1 = "Message#EMPTY_INFO_TAGLIB");
+		}
+
+		task ("Assert that it's possible to publish it") {
+			PublicationsNavigator.gotoReviewChanges();
+
```

Comment:
Can you add the steps to review the changes under Asset List Entry, you can toggle Show All Items to make them show up.